addon/models/osf-model.js

```diff
@@ -17,6 +17,7 @@ import HasManyQuery from 'ember-data-has-many-query';
 export default DS.Model.extend(HasManyQuery.ModelMixin, {
     links: DS.attr('links'),
     embeds: DS.attr('embed'),
+    errors: DS.attr(),
```

Comment:
What about getting errors on relationships?  If I fetch node.parent and there is no parent, or I don't have permission to view the parent?